paasta_tools/cli/cmds/status.py

```diff
@@ -125,6 +125,13 @@ def add_subparser(subparsers,) -> None:
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    status_parser.add_argument(
+        "--new",
```

Comment:
nit: `--experimental` instead? `--new` feels like it applies officially released and supported.

Reply:
i guess id rather not given that i'd have to change this all over the place

Reply:
This would be a bit of a pain to change at this point, so I am going to skip this unless you feel strongly